index.html

```diff
@@ -41,23 +53,300 @@ <h3>LET'S CONNECT</h3>
             </li>
+          </ul>
+          <div class="textTonic">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="language">
+            <ul>
+              <li>
+                <p>HTML</p>
+              </li>
+              <li>
+                <p>CSS</p>
+              </li>
+              <li>
+                <p>Javascript</p>
+              </li>
+            </ul>
+          </div>
+          <div class="seeMore">
+            <h3>See project</h3>
+          </div>
+        </div>
+      </div>
+
+      <div class="card-four">
+        <div class="card-one">
+          <img src="Icon and img/history Portfolio.png" alt="" />
+          <h3>Multi-Post Stories</h3>
+          <ul>
             <li>
-            <a href="https://www.linkedin.com/in/tchasinga-jacques-76aba7214/"><img src="img/Linkedin icon.png" alt=""></a> 
+              <h4>Canopy</h4>
             </li>
             <li>
-             <a href="https://medium.com/@tchasingajacques"> <img src="img/Vector.png" alt=""></a>  
+              <img src="Icon and img/Counter.png" alt="" />
             </li>
             <li>
-                <a href="https://github.com/tchasinga" target="_blank" rel="noopener noreferrer"><img src="img/Github.png" alt=""></a>
+              <p>Band End Dev</p>
             </li>
             <li>
-               <a href="https://wellfound.com/jobs"><img src="img/welldone.png" alt=""></a> 
+              <img src="Icon and img/Counter.png" alt="" />
             </li>
+            <li>
+              <p>2015</p>
+            </li>
+          </ul>
+          <div class="textTonic">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="language">
+            <ul>
+              <li>
+                <p>HTML</p>
+              </li>
+              <li>
+                <p>CSS</p>
+              </li>
+              <li>
+                <p>Javascript</p>
+              </li>
+            </ul>
+          </div>
+          <div class="seeMore">
+            <h3>See project</h3>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="aboutme">
+        <h1>About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <h4 class="connect">LET'US CONNECT</h4>
+      </div>
+      <div class="contactLink">
+        <ul>
+          <li>
+            <a href="https://twitter.com/TchasingaJacque"></a>
+            <img src="img/Twitter.png" alt="" />
+          </li>
```

Comment:
- [ ] Dear @tchasinga, Here we need to see every icon as a link, Hence every icon should be wrapped inside the `a` tag. That will make your icon a link rather than just another image. Please wrap your icons in the `a` tag. 
> something like ...
```html 
 <li>
    <a href="https://twitter.com/TchasingaJacque"><img src="img/Twitter.png" alt="" /></a>
  </li>
```